doc/source/walkthrough.rst

```diff
@@ -39,150 +54,225 @@ To start a multi-node Ray cluster, see the :ref:`cluster setup page <cluster-ind
 Remote functions (Tasks)
 ------------------------
 
-Ray enables arbitrary Python functions to be executed asynchronously. These asynchronous Ray functions are called "remote functions". The standard way to turn a Python function into a remote function is to add the ``@ray.remote`` decorator. Here is an example.
+Ray enables arbitrary functions to be executed asynchronously. These asynchronous Ray functions are called "remote functions". Here is an example.
 
-.. code:: python
+.. tabs::
+  .. group-tab:: Python
 
-    # A regular Python function.
-    def regular_function():
-        return 1
+    .. code:: python
 
-    # A Ray remote function.
-    @ray.remote
-    def remote_function():
-        return 1
+      # A regular Python function.
```

Comment:
I reorganized this part. I used to compare regular python function and remote function in different aspects. Now it just directly explains the features of remote functions. 

This is to make Python and Java examples more aligned. Because in Java, there is no such an entity of "remote function". You just take a regular Java function, and invoke it by `Ray.task(func).remote()`

I think the comparison was unnecessary. Because all users should already know regular Python functions are called by `regular_function()`.